content/blogposts/2022/release-3.41.md

```diff
@@ -0,0 +1,63 @@
+    url: https://github.com/sourcegraph/sourcegraph/pull/35319
+    category: Batch Changes
+  - description: Added `sort` and `limit` filters for capture group insights. This gives Code Insights users more control over which series are displayed.
+    url: https://github.com/sourcegraph/sourcegraph/pull/34611
+    category: Code Insights
+  - description: 'Commit and diff search: The hard limit of 50 repositories has been removed, so you can now run broader searches. Long-running searches will continue running until the timeout is hit.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/36486
+    category: Search
+  - description: 'Zoekt-indexserver has a new debug landing page, /debug, which exposes information about the queue, the list of indexed repositories, and the list of assigned repositories for easier visual debugging. Admins can reach the debug landing page by going to the site admin view and selecting Instrumentation > indexed-search-indexer > Debug.'
+    url: https://github.com/sourcegraph/zoekt/pull/346
+    category: Admin
+---
+ 
+Sourcegraph 3.41 is now available! Here are some highlights from this release:
+
+## Running batch changes server-side is moving to beta
+
+Until now, you had to run `src-cli` locally to create your batch changes. `src-cli` pulls repo archives through Sourcegraph, and runs your batch spec to create diffs. For large amounts of repositories or resource-intensive code rewrites, running `src-cli` locally could take an impractical amount of time or be brittle.
+
+Today, we're announcing that you can now run [batch changes server-side (beta)](https://docs.sourcegraph.com/batch_changes/explanations/server_side). 
+
+Setting up running batch changes server-side allows you to:
+- Run large-scale or resource intensive batch changes without clogging your local machine.
+- Run large batch changes faster by distributing them across an autoscaled pool of compute instances.
+- Get a better debugging experience, with logs being streamed directly into Sourcegraph.
+- Simplify setup for local users (no local `src-cli`) by having the Sourcegraph admin manage server-side setup.
+
+This is the outcome of months of iteration on the experimental version of the feature, and we're grateful to early users for their feedback!
+
+This feature requires admins to setup [executors](https://docs.sourcegraph.com/admin/executors) (much like CI agents), which Sourcegraph will use to offload expensive tasks. Executors can also be used to run Code Intelligence [auto-indexing](https://docs.sourcegraph.com/code_intelligence/how-to/enable_auto_indexing) (experimental).
+
+##  Batch specs can now mount local files on steps containers (experimental)
+
+As we improve Batch Changes, we're heavily focusing on making it easy to write and debug batch specs. Iteration time needs to be as small as possible, so that users can quickly put together a spec, run it, see results, and quickly improve on it.
```

Comment:
```suggestion
As we improve Batch Changes, we're focused on making it easy to write and debug batch specs. Iteration time needs to be as small as possible, so that users can quickly put together a spec, run it, see results, and quickly improve on it.
```